sklearn/manifold/locally_linear.py

```diff
@@ -190,10 +189,9 @@ def locally_linear_embedding(
 
     Parameters
     ----------
-    X : {array-like, sparse matrix, BallTree, KDTree, NearestNeighbors}
+    X : {array-like, BallTree, KDTree, NearestNeighbors}
```

Comment:
I can confirm this is not working for me with a *Tree, but is working with a NearestNeighbors. Surely we should have a test to that effect?